### DISCUSSION TITLE
Simd support for WASM

### DISCUSSION THREAD

**Discussion by kalwalt** [titzer/virgil#234] [Q&A] [answered]
I saw this [issue](https://github.com/titzer/wizard-engine/issues/5) in the wizard-engine and i'm asking if there is some example for this. Any help will be appreciated!

**Comment by titzer** [comment, marked as answer]
Right now Virgil doesn't support SIMD in the source language.

**Reply by kalwalt** [reply]
Ok, thank you for the answer!
